chore(blog): add hdi interview blog (#3575)
asyncapi/website: markdown/blog/hdi-global-interview.md

---
title: "Interview with Manuel Ottlik, Product Owner at HDI Global SE: Leveraging AsyncAPI for Integration Transparency!"

> date: 2025-01-20T06:00:00+01:00

Author: We are in 2025 already.

type: Communication
tags:

>   - Project Status

Reviewer: Change the tags here, as this is not a Project Status article

Author: Okay

cover: /img/posts/marketing-images/hdi-interview.webp
authors:
  - name: Atinuke Oluwabamikemi Kayode
    photo: /img/avatars/bami.webp
    link: https://www.linkedin.com/in/atinuke-oluwabamikemi-kayode-5b838b1b7/
    byline: AsyncAPI Community Marketing Specialist
excerpt: 'This is an interview session with Manuel, the Product Owner at HDI Global SE explaining how they use AsyncAPI to maintain their API catalog.'
---

# Interview with Manuel Ottlik, Product Owner at HDI Global SE: Leveraging AsyncAPI for Integration Transparency!

> HDI Global SE, a leading player in the insurance industry, has become our first end-user, a non-software vendor company that not only openly admits they use AsyncAPI and share their use case but also became official sponsors and a valuable partner of the AsyncAPI Initiative. Their commitment to open source is evident not only through financial support but also through active contributions and the adoption of AsyncAPI to enhance transparency in their integration platform. We're ecstatic to have Manuel Ottlik, Product Owner of the Global Integration Platform at HDI Global SE, as one of our dedicated AsyncAPI ambassadors. Manuel has played a key role in driving the adoption of AsyncAPI at HDI, transforming our partnership into a meaningful use case.

Reviewer 2: ```suggestion
HDI Global SE, a leading player in the insurance industry, has become our first end-user, a non-software vendor company that not only openly admits they use AsyncAPI and share their use case but also became our official sponsors and a valuable partner of the AsyncAPI Initiative. Their commitment to open source is evident not only through financial support but also through active contributions and the adoption of AsyncAPI to enhance transparency in their integration platform. We are ecstatic to have Manuel Ottlik, Product Owner of the Global Integration Platform at HDI Global SE, as one of our dedicated AsyncAPI ambassadors. Manuel has played a key role in driving the adoption of AsyncAPI at HDI, transforming our partnership into a meaningful use case.
```

> HDI Global SE, a leading player in the insurance industry, has become our first end-user, a non-software vendor company that not only openly admits they use AsyncAPI and share their use case but also became our official sponsors and a valuable partner of the AsyncAPI Initiative. Their commitment to open source is evident not only through financial support but also through active contributions and the adoption of AsyncAPI to enhance transparency in their integration platform. We are ecstatic to have Manuel Ottlik, Product Owner of the Global Integration Platform at HDI Global SE, as one of our dedicated AsyncAPI ambassadors. Manuel has played a key role in driving the adoption of AsyncAPI at HDI, transforming our partnership into a meaningful use case.

Reviewer: ```suggestion
HDI Global SE, a leading player in the insurance industry, has become our first end-user, a non-software vendor company that not only openly admits they use AsyncAPI and share their use case but also became our official sponsor and a valuable partner of the AsyncAPI Initiative. 
Their commitment to open source is evident through financial support, active contributions, and adoption of AsyncAPI to enhance transparency in their integration platform. We are ecstatic to have Manuel Ottlik, Product Owner of the Global Integration Platform at HDI Global SE, as one of our dedicated AsyncAPI ambassadors. Manuel has played a key role in driving the adoption of AsyncAPI at HDI, transforming our partnership into a meaningful use case.
```

> HDI Global SE, a leading player in the insurance industry, has become our first end-user, a non-software vendor company that not only openly admits they use AsyncAPI and share their use case but also became our official sponsors and a valuable partner of the AsyncAPI Initiative. Their commitment to open source is evident not only through financial support but also through active contributions and the adoption of AsyncAPI to enhance transparency in their integration platform. We are ecstatic to have Manuel Ottlik, Product Owner of the Global Integration Platform at HDI Global SE, as one of our dedicated AsyncAPI ambassadors. Manuel has played a key role in driving the adoption of AsyncAPI at HDI, transforming our partnership into a meaningful use case.

Reviewer: Would be great to link Manuel's socials

Author: Yeah..

*However, we realized that to fully phase out the enterprise service bus, we’d need both synchronous and asynchronous communication capabilities. This led us to incorporate asynchronous services through an event broker, ensuring comprehensive integration support.*

## Key Challenges in Merging Technologies

We delved deeper into the complexities of integrating various technologies.

**Bami: Q: As Product Owner of the Global Integration Platform, what key challenges have you faced in merging technologies like a service bus, API management, and an event broker into a unified platform?**

***Manuel:** One of the key challenges is that these technologies are often introduced at different times. For example, most companies start with API management and later realize the need for asynchronous capabilities. As a result, you end up with distinct interfaces for asynchronous and synchronous communication, even though they represent the same business entities.*

> *The challenge is merging these technologies and their interfaces cohesively. At HDI, we manage the platform, not the interfaces, but we issue guidance on building interfaces. We aim to have a consistent view of business objects, regardless of the integration product being used. The schemas of these business objects should align, whether they’re using OpenAPI or AsyncAPI.*

Reviewer: ```suggestion
*The challenge is merging these technologies and their interfaces cohesively. At HDI, we manage the platform, not the interfaces, but we issue guidance on building interfaces. We aim to have a consistent view of business objects, regardless of the integration product being used. The schemas of these business objects should align whether they’re using OpenAPI or AsyncAPI.*
```

## HDI’s Contributions to Open Source

**Bami: How does HDI Global SE contribute to the open-source community, and what benefits has the company seen from these contributions?**

***Manuel:** We contribute in various ways. I participate in the CNCF Serverless Working Group that is building CloudEvents and xRegistry, attend meetings, and we have created a case study for AsyncAPI. We’ve also sponsored your project and actively participate in open source projects through code contributions when needed.*

## HDI Global SE’s Interest in AsyncAPI

We then discussed HDI’s interest in AsyncAPI.

**Bami: What sparked HDI Global SE’s interest in AsyncAPI?**

***Manuel:** Our interest in AsyncAPI stems from our goal of transparency. We maintain an API catalog for synchronous and asynchronous communication, with OpenAPI covering the former. AsyncAPI fills the gap for the asynchronous world, providing the same transparency for service-based topics and messages.*

**Bami: How do you see AsyncAPI playing a role in HDI Global SE’s integration strategy, especially as the company continues its cloud adoption?**

> ***Manuel:** It's the same reason; it's going to be the part that enables transparency over asynchronous interfaces, service bus topics and messages in them, event brokers and events that get sent over there.*

Reviewer: ```suggestion
***Manuel:** It's the same reason; it's going to be the part that enables transparency over asynchronous interfaces, service bus topics and messages in them, event brokers, and events that get sent over there.*
```

Wrapping up, we explored Manuel’s thoughts on broader industry trends.

**Bami: What emerging trends in API management and cloud integration are you most excited about?**

> ***Manuel:** Asynchronous communication is still developing but will eventually be on par with synchronous communication in enterprise integration. I’m particularly excited about xRegistry, which will bridge the gap between synchronous and asynchronous communication by providing a standard for schema registries that can then host all business objects and their schemas as well as the interface descriptions that are built upon those schemas.*

Reviewer: ```suggestion
***Manuel:** Asynchronous communication is still developing but will eventually be on par with synchronous communication in enterprise integration. I’m particularly excited about xRegistry, which will bridge the gap between synchronous and asynchronous communication by providing a standard for schema registries that can then host all business objects and their schemas, as well as the interface descriptions that are built upon those schemas.*
```

---
title: "Interview with Manuel Ottlik, Product Owner at HDI Global SE: Leveraging AsyncAPI for Integration Transparency!"
date: 2025-01-20T06:00:00+01:00
type: Communication
tags:
  - Interview
  - Case Study
  - Community
cover: /img/posts/marketing-images/hdi-interview.webp
authors:
  - name: Atinuke Oluwabamikemi Kayode
    photo: /img/avatars/bami.webp
    link: https://www.linkedin.com/in/atinuke-oluwabamikemi-kayode-5b838b1b7/
    byline: AsyncAPI Community Marketing Specialist
excerpt: 'Learn how HDI Global SE, our first end-user and silver sponsor, leverages AsyncAPI for integration transparency. Manuel Ottlik shares insights on merging enterprise service buses with modern API management and event-driven architectures.'
---

# Interview with Manuel Ottlik, Product Owner at HDI Global SE: Leveraging AsyncAPI for Integration Transparency!

> HDI Global SE, a leading player in the insurance industry, has become our first end-user, a non-software vendor company that not only openly admits they use AsyncAPI and share their use case, but also became official sponsor and a valuable partner of the AsyncAPI Initiative.

Reviewer 3: @iambami Is this suggestion valid?
